Makefile

```diff
@@ -314,7 +314,7 @@ tianocore_fd: _tianocore_prepare
 tianocore_img: _check_atf_tools _check_atf_slim _check_board_setting tianocore_fd
 	@echo "Build Tianocore $(BUILD_VARIANT_UFL) Image - ATF VERSION: $(ATF_MAJOR).$(ATF_MINOR).$(ATF_BUILD)..."
 	$(eval DBB_KEY := $(EDK2_PLATFORMS_SRC_DIR)/Platform/Ampere/$(BOARD_NAME_UFL)Pkg/TestKeys/Dbb_AmpereTest.priv.pem)
-	@dd bs=1024 count=2048 if=/dev/zero | tr "\000" "\377" > $(OUTPUT_RAW_IMAGE)
+	@dd bs=1024 count=2048 if=/dev/zero | LC_ALL=C tr "\000" "\377" > $(OUTPUT_RAW_IMAGE)
```

Comment:
Should we verify the machine type and set LC_ALL=C globally instead of specifying it in every 'tr' command?

Reply:
I'm not sure it matters either way. If you think that's cleaner, feel free to update my PR.